MiniTwitch.Irc/Models/Privmsg.cs

```diff
@@ -152,7 +152,7 @@ internal Privmsg(ReadOnlyMemory<byte> memory, IrcClient? source = null)
 
                 //vip
                 case 335:
-                    vip = TagHelper.GetBool(tagValue);
+                    vip = true;
```

Comment:
> A Boolean value that determines whether the user that sent the chat is a VIP. The message includes this tag if the user is a VIP; otherwise, the message doesn’t include this tag (check for the presence of the tag instead of whether the tag is set to true or false).

https://dev.twitch.tv/docs/irc/tags/#privmsg-tags